pilot/pkg/proxy/envoy/v2/eds_test.go

```diff
@@ -769,6 +766,7 @@ func addEdsCluster(server *bootstrap.Server, hostName string, portName string, a
 		},
 	})
 	server.EnvoyXdsServer.Push(&model.PushRequest{Full: true})
+	time.Sleep(1 * time.Second)
```

Comment:
same confusion

Reply:
please do not sleep, it will flake in CI

Reply:
yeah. we are experimenting. will remove in the final version

Reply:
same explanation as https://github.com/istio/istio/pull/20020#discussion_r365498604